src/main/webapp/item-page.html

```diff
@@ -2,37 +2,65 @@
 <html>
   <head>
     <meta charset="UTF-8" />
-    <title>Item Page</title>
-    <!-- Bootstrap CSS -->
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Entertainment Hub - Item Page.</title>
     <link
       rel="stylesheet"
-      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
     />
-    <link rel="stylesheet" href="style.css" />
-    <script src="script.js"></script>
-    <script src="itempagescript.js"></script>
-    <!-- Load an icon library -->
     <link
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
-    <!-- Javascript for Bootstrap -->
+    <link rel="stylesheet" href="style.css" />
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+    <script src="itempagescript.js"></script>
+    <script src="script.js"></script>
   </head>
 
   <body onload="loadItemPage()">
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark mb-5">
+      <div class="container">
+        <a href="#" class="navbar-brand">Entertainment Hub </a>
+        <a href="/index.html"><i class="fa fa-home fa-2x"></i></a>
+      </div>
+    </nav>
+
     <div class="container">
-      <div id="item-container"></div>
+      <div
+        class="card"
+        style="width: 27em; margin: 0 auto;"
+        style="height: 20rem;"
+      >
+        <div id="item-container"></div>
+      </div>
+      <br />
       <footer>
-        <ul id="comment-container"></ul>
-        <form>
-          <textarea name="comment"></textarea>
-          <br /><br />
-          <button onclick="sendFormData()">Submit</button>
-        </form>
+        <br />
+        <h2>Comments</h2>
+        <ul class="list-group" id="comment-container"></ul>
+        <div id="comments">
+          <br />
+          <form>
+            <h4>Enter a comment below</h4>
+            <textarea id="comment"></textarea>
+            <br /><br />
```

Comment:
I think it's probably a good idea to start getting rid of the ```<br>``` tags and instead use the spacing classes given by Bootstrap: https://getbootstrap.com/docs/4.0/utilities/spacing/